app/src/main/java/com/upf/memorytrace_android/util/MemoryTraceConfig.kt

```diff
@@ -83,7 +85,11 @@ object MemoryTraceConfig {
             )
         }
 
+    var theme: Int
+        get() = pref.getInt(THEME, ThemeUtil.ThemeMode.DARK.ordinal)
+        set(value) = pref.edit { putInt(THEME, value) }
+
```

Comment:
Int 말고 String 그대로 저장하면 어떨까요 ?
이것만 보면 Int 가 ordinal 을 나타내고 있는지 몰라서 추후 혼동이 있을 것 같습니다.
```suggestion
    var theme: String
        get() = pref.getString(THEME, ThemeUtil.ThemeMode.DARK.name)
        set(value) = pref.edit { putInt(THEME, value) }
```

Reply:
엇 네네, 이거 키값 빼면서 수정을 안했네요 ㅜㅜ 👍👍👍